lib/js/stanford_events.behavior.js

```diff
@@ -42,19 +45,34 @@ export default {
         $('section.event').attr('data-end-date');
 
       // Build the labels and sentence
-      const pastLabel =
+      // Has type at top
+      const eventTypeLabel =
         $('<span class="su-event-label-past">' + Drupal.t('Past Event') +
           '<span class="divider">' + Drupal.t('|') + '</span></span>');
 
+      // Has no type at top
+      const eventLabel =
+        $('<span class="su-event-label-past">' + Drupal.t('Past Event') +
+          '</span></span>');
+
       const pastText =
         $('<div class="su-event-text-past">' +
           Drupal.t('This event has passed.') + '</div>');
 
       // Apply past label and text
-      if (eventDate < todayBegin) {
-        pastLabel.prependTo(eventType.first());
+      if (eventDate < todayBegin && eventTypeLabelLoc.length) {
+        // Top label with a Type
+        eventTypeLabel.prependTo(eventTypeLabelLoc.first());
         pastText.appendTo(eventTextLocation.last());
       }
+      else {
+        // Top label without a Type
+        if (eventDate < todayBegin && (eventLabel.length)) {
```

Comment:
`eventLabel` is always going to be true since you are setting the variable [above](https://github.com/SU-SWS/stanford_events/pull/41/files#diff-93ca905e8ac609065e4a0e78520065bbf6406bf1ab980c72b139361fb464204bR54)